.travis.yml

```diff
@@ -11,6 +11,8 @@ before_script:
   - psql -c "CREATE ROLE authorsHaven WITH SUPERUSER LOGIN PASSWORD 'authorshaven';" -U postgres
   - psql -c "CREATE DATABASE testAuthorsHaven;" -U postgres
   - export DJANGO_SETTINGS_MODULE=authors.settings
+  - export SECRET_KEY="secretkey"
```

Comment:
@meshack-mbuvi push these to the environment.

Reply:
I have done it now. Please confirm that it has been done. Thank you.